### ISSUE TITLE
Hydra can't open tx textures

### ISSUE THREAD

**Issue by sirpalee** [PixarAnimationStudios/USD#97]
Hydra can't open tx textures. It's commonly used, so it might be a good idea just to enable it. It's just adding the tx extension to the supported formats by Glf_OIIOImage.

```

---
 pxr/imaging/lib/glf/plugInfo.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pxr/imaging/lib/glf/plugInfo.json b/pxr/imaging/lib/glf/plugInfo.json
index ce86c17..bb5a20f 100644
--- a/pxr/imaging/lib/glf/plugInfo.json
+++ b/pxr/imaging/lib/glf/plugInfo.json
@@ -6,7 +6,7 @@
                 "Types": {                                                                  
                     "Glf_OIIOImage" : {
                         "bases": ["GlfImage"],
-                        "imageTypes": ["bmp", "exr", "jpg", "png", "tif", "zfile"],
+                        "imageTypes": ["bmp", "exr", "jpg", "png", "tif", "zfile", "tx"],
                         "precedence": 0
                     },
                     "GlfPtexTexture" : {
-- 
2.7.4
```

**Comment by jtran56** [comment]
Filed as internal issue #139514.

**Comment by meshula** [comment]
This one is interesting. Wouldn't it make sense for glf to ask the linked OIIO for a list of readable imageTypes, instead of hard coding it in the plugInfo.json file?

**Comment by sirpalee** [comment]
@meshula Right now hydra doesn't support all the data kinds that oiio does (like field3d), and also ptex is implemented in a separate class. So you'll end up manually specifying the formats you are interested in (or excluding some of them), like now.

**Comment by meshula** [comment]
Makes sense. Thanks

**Comment by c64kernal** [comment]
Thanks yeah -- and there's a bit of a chicken and egg problem where OIIO is implemented as a "plugin" in our system that we would (in theory) avoid loading altogether until we knew we needed it (based on matching manually listed extensions in the plugInfo file).  However, for now that isn't being taken advantage of, of course, hence the word plugin in quotes :)

**Comment by davidgyu** [comment]
One bit of confusion is that both ".tx" and ".tex" filename extensions have been used to designate files created by prman's 'txmake' using prman's proprietary texture image format. OIIO has support for ".tx" files compatible with OIIO's 'maketx'.

We use additional glfImage plugins within the studio to load proprietary prman texture image files registered for both ".tx" and ".tex" extensions.

We should be able to support both and make sure that we can load all of the compatible formats supported by the local OIIO installation.  Taking a look...

**Comment by sunyab** [comment]
This was fixed by pull request #467, which was merged and released in 0.8.5. Closing this out, thanks!
